weblate/machinery/microsoft.py

```diff
@@ -113,13 +122,17 @@ def download_languages(self):
     def download_translations(self, source, language, text, unit, user):
         """Download list of possible translations from a service."""
         args = {
-            'text': text[:5000],
+            'api-version': '3.0',
             'from': source,
             'to': language,
-            'contentType': 'text/plain',
             'category': 'general',
         }
-        response = self.request("get", TRANSLATE_URL, params=args)
+        response = self.request(
+            "post",
+            TRANSLATE_URL,
+            params=args,
+            json=[{'Text': text[:5000]}]
+        )
```

Comment:
Seems like some payload processing is missing here as it ends up being displayed this way:

![image](https://user-images.githubusercontent.com/212189/76070279-89ce0400-5f94-11ea-9ce1-1628267bb44c.png)

I've extended the tests to cover this in f4226ace09545d8fb74f1867ad25115903e658fd so please rebase on current master.

Reply:
Sure, can you point me somewhere how should i format the payload?

Thanks in avance!